db/seeds.rb

```diff
@@ -32,6 +33,8 @@
 
       event = Event.create!(name: event_data["title"], date: event_data["published_at"], organisation: organisation)
 
+      event.update!(slug: event_data["slug"])
+
```

Comment:
The slug is needed here so the slugs from the `data/organisations.yml` and the `data/helveticruby/playlists.yml` files are actually respected.